doc/changing_derivatives.md

```diff
@@ -34,7 +37,12 @@ end
 *Scenario: Your app is currently working only with original files, and you want
 to introduce derivatives.*
 
-Start by loading the `derivatives` plugin and defining your processing logic:
+#### 1. Add `derivatives` plugin and `image_processing` gem
```

Comment:
I would rather that this guide focused on what to do _after_ a change in processing was made. This is how other sections in this guide are structured, and it makes sense to me to have it more conversational, because this guide covers several scenarios.